keygen/src/auth.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Xaptum, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _XCR_COMM_H
+#define _XCR_COMM_H
+
+
+#include <jansson.h>
+
+struct xcr_session {
+    char username[255];
+    char token[512];
+};
+
+struct xcr_login {
```

Comment:
Maybe `auth_login` instead of `xcr_login`, so that the prefix is consistent within this file?  Ditto for the functions below:  `xcr_send_login` -> `auth_login`, etc.  The